style-guides/scss.md

```diff
@@ -0,0 +1,123 @@
+<script>{
+	"title": "Chassis SCSS Style Guide"
+}</script>
+
+This page outlines the style guide for SCSS for the jQuery Chassis project.
+
+## Comments
+
+SCSS comments should follow the valid CSS comment guidelines outlined in the jQuery contribute CSS style guide.
```

Comment:
There was no discussion about this on that date in the meeting or in any meeting on any date actually ( at least this year i didn't check last year but this came up much after that ). See http://irc.jquery.org/%23jquery-meeting/default_%23jquery-meeting_20150505.log.html

The discussion on this actually took place in the Typography PR here https://github.com/jquery/css-chassis/pull/57#discussion_r29560502 and you will see that the conclusion before merging was that we should use whatever makes sense which was no change from what was in the pr / we were already doing.

As for the PR that updated the comments. It does not look like that or the issue it fixes, was ever reviewed or commented on by any one before it was merged / fixed and both reference their reason as the same meeting you reference above which contains no discussion of this. ( There was also no discussion about it in #css-chassis at any point that i can find in the logs. )